### PR TITLE
chore(release): cut new major version

### PR DESCRIPTION
**Pull request by varl**
All things from this issue: https://github.com/dhis2/app-platform/pull/645 are in the `beta` branch.

### CODE REVIEW

**Review by mediremi — APPROVED** [review]

**Comment by varl** [comment]
We want to get this in: https://github.com/dhis2/app-runtime/pull/820 + additional work required in app-platform.

And then bump the deps in app-platform before merging this.

**Review by ismay — APPROVED** [review]

**Comment by varl** [comment]
Will merge after https://github.com/dhis2/app-platform/pull/549.

**Pull request merged**
